lib/agenda/stop.js

```diff
@@ -18,8 +18,21 @@ module.exports = function(cb) {
   const _unlockJobs = function(done) {
     debug('Agenda._unlockJobs()');
     const jobIds = self._lockedJobs.map(job => job.attrs._id);
-    debug('about to unlock jobs with ids: %O', jobIds);
-    self._collection.updateMany({_id: {$in: jobIds}}, {$set: {lockedAt: null}}, done);
+
+    if (jobIds.length >= 1) {
+      debug('about to unlock jobs with ids: %O', jobIds);
+      self._collection.updateMany({_id: {$in: jobIds}}, {$set: {lockedAt: null}}, err => {
+        if (err) {
+          return done(err);
+        }
+
+        self._lockedJobs = [];
```

Comment:
Is there a test case for this change? (If it even is a change, but looks like previously it would just leave this array behind).